apps/builder/app/canvas/features/webstudio-component/webstudio-component.tsx

```diff
@@ -183,12 +183,19 @@ export const WebstudioComponentDev = ({
     onClick: (event: MouseEvent) => {
       event.preventDefault();
       if (event.currentTarget instanceof HTMLAnchorElement) {
+        // @todo use Navigation API once implemented everywhere
+        // https://developer.mozilla.org/en-US/docs/Web/API/Navigation_API
         handleLinkClick(event);
+      } else if (typeof userProps.onClick === "function") {
```

Comment:
When is this the case? When the element is not a link and there is an onClick callback? What happens if its a link, can't it have an onClick?

Reply:
Bypass event handlers to other components for example Button